client/src/components/CreatePost.jsx

```diff
@@ -1,33 +1,87 @@
 import React from 'react';
 import { Card, Icon, Button, Label, Comment, Input } from 'semantic-ui-react';
 import axios from 'axios';
+import Dropzone from 'react-dropzone';
 
 class CreatePost extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      postText: ''
+      postText: '',
+      accepted: [],
+      rejected: []
     }
   }
-  createPost() {
-    let postInput = document.getElementById('postInput').value.replace(`'`, `''`);
-    let username = this.props.name;
-    document.getElementById('postInput').value = '';
-    
+
+  handlePostInput(event) {
+    const value = event.target.value;
+
     this.setState({
-      postText: postInput
-    })
-    axios.post(`/api/${username}/posts`, { 'text': postInput })
-      .then((res) => {
-        this.props.getAllPosts ? this.props.getAllPosts() : this.props.renderNewPost(this.props.name);
+      postText: value
+    });
+  }
+
+  onDrop(accepted, rejected) {
+    this.setState(
+      { accepted, rejected }
```

Comment:
Unfamiliar with this syntax. Is this destructuring?

Reply:
I think it's actually object literal property notation: 
https://ariya.io/2013/02/es6-and-object-literal-property-value-shorthand